lib/tasks/import_countries.rake

```diff
@@ -1,26 +1,29 @@
 require 'csv'
 
 namespace :import do
-  desc "import countries data into database"
-  task :countries => [:environment] do
+  desc 'import countries data into database'
+  task countries: [:environment] do
+    # Strip out accents, commas, apostrophes, brackets as they mess with the URL
+    def sanitize(name) 
+      I18n.transliterate(name).downcase.gsub(/[,'()]/, '').titleize
+    end
```

Comment:
Does this mean we don't need the pathname gsub going on somewhere else in this PR?

Reply:
I'll have to take a look to see if any refactoring can be done in the helpers

Reply:
No, we still need that method, as that basically kebab-cases the country name. It's true that we don't need to account for apostrophes any more though.